packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.tsx

```diff
@@ -113,17 +113,17 @@ function useDateRangeCalendarDefaultizedProps(
 ): DateRangeCalendarDefaultizedProps {
   const utils = useUtils();
   const defaultDates = useDefaultDates();
-  const defaultReduceAnimations = useDefaultReduceAnimations();
   const themeProps = useThemeProps({
     props,
     name,
   });
+  const reduceAnimations = useReduceAnimations(themeProps.reduceAnimations);
```

Comment:
I unified the behavior of `useReduceAnimations` with `usePickerOrientation` it now takes the explicit value as a param.